### DISCUSSION TITLE
Fast Boot in RaspAP mode with RPi Zero

### DISCUSSION THREAD

**Discussion by Karthikai-28** [RaspAP/raspap-webgui#844] [Q&A]
Currently, I am using RaspAP with RPi Zero, while booting up raspapd.service alone takes 38.341s. If I consider all other services, it goes more than 70 seconds to boot up the Pi. Is there any way to fast boot the RPi zero while RaspAP is running?

**Comment by billz** [comment]
Those values are unusually high. How did you arrive at them?
I don't have a Zero handy at the moment, but here are RPi4 numbers for comparison:

```
$ systemd-analyze blame
12.393s raspapd.service
7.004s hciuart.service
3.984s docker.service
3.651s networking.service
2.034s dnsmasq.service
1.928s ifupdown-pre.service
1.646s dev-mmcblk0p2.device
1.619s containerd.service
1.569s apt-daily-upgrade.service
1.475s apt-daily.service
1.331s man-db.service
1.073s lighttpd.service
```

12 seconds is realistic as raspap.service manages 3 other network-related services, with a configurable delay between each one. If a faster boot is your priority, you can disable this service and manage them yourself, at the expense of not having `uap0` available at boot. See the docs for more info.
